pkg/controller/garbagecollector/operations.go

```diff
@@ -30,13 +30,30 @@ import (
 	"k8s.io/kubernetes/pkg/client/retry"
 )
 
+type restMappingError struct {
+	kind    string
+	version string
+}
+
+func (r *restMappingError) Error() string {
```

Comment:
Mind making a short `Error` and a long `Message` or some such?  The detail is great in a log, less great if it ever escaped.